src/utils.js

```diff
@@ -19,9 +21,9 @@
     };
 
     Utils.prototype.genRandomString = function () {
```

Comment:
I wonder why we kept this function in the end? I'm not sure but I thought Mongo would automatically generate an ObjectId string if the `_id` field was not passed. Haven't tried this myself but if that's the case then I don't see the need to keep this function around.